.github/workflows/worker-deploy.yml

```diff
@@ -0,0 +1,29 @@
+name: Deploy
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    name: Deploy
+    steps:
+      - name: Setup Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: "20"
+      - uses: actions/checkout@v4
+      - uses: cloudflare/wrangler-action@v3
+        with:
+          wranglerVersion: "3.57.0"
+          apiToken: ${{ secrets.CLOUDFLARE_API_TOKEN }}
+          secrets: |
+            SUPABASE_URL
+            SUPABASE_KEY
+            UBIQUIBOT_TOKEN
+        env:
+          SUPABASE_URL: ${{ secrets.SUPABASE_URL }}
+          SUPABASE_KEY: ${{ secrets.SUPABASE_KEY }}
+          UBIQUIBOT_TOKEN: ${{ secrets.UBIQUIBOT_TOKEN }}
```

Comment:
@Keyrxng `UBIQUIBOT_TOKEN` is the github app token, right? Isn't it supposed to be passed in a plugin invocation POST request (assuming we're running the plugin as a Cloudflare worker)?

Reply:
I'll resolve this as @gentlementlegen has answered already pretty much